# This is a list of histories of some robots
bool comGraphEvent
int16[] robList
History[] listOfH
